README.md

```diff
@@ -261,6 +264,31 @@ DESCRIPTION
 
 _See code: [src/commands/app/build.js](https://github.com/adobe/aio-cli-plugin-app/blob/13.3.0/src/commands/app/build.js)_
 
+## `aio app clean-build`
```

Comment:
just checking: this was not generated manually, correct? Usually before publishing, it will generate this via the `prepack` npm script (which runs on `npm pack`). You can run this manually of course as well.

Reply:
I ran the prepack locally to generate the README.md content for the new command. Should I revert the REAME in that case and let the publish take care of it, in that case?